tests/worker_coordinator/runner_test.py

```diff
@@ -2801,6 +2801,164 @@ async def test_query_vector_search_with_custom_id_field_inside_source(self, open
             headers={"Accept-Encoding": "identity"}
         )
 
+    @mock.patch('osbenchmark.client.RequestContextHolder.on_client_request_end')
+    @mock.patch('osbenchmark.client.RequestContextHolder.on_client_request_start')
+    @mock.patch("opensearchpy.OpenSearch")
+    @run_async
+    async def test_query_vector_radial_search_with_min_score(self, opensearch, on_client_request_start, on_client_request_end):
```

Comment:
can you add test for if both k and min/max is provided?

Reply:
Added tests